website/content/docs/setup/vault-configuration/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Vault Configuration
+linktitle: Vault Configuration
+weight: 30
+---
+
+The {{< name >}} uses [HashiCorp Vault](https://www.vaultproject.io/) to provide [Modules](../../overview/modules) running in the remote clusters with the dataset credentials to execute their tasks.
+
+This page contains two sections:
+
+1. [This section](#configure-vault-kubernetes-auth-method-in-each-cluster) contains the required steps to enable the modules to authenticate to Vault in order for them to retrieve the dataset credentials.
+
+2. [This section](#add-vault-secret-plugin) describes how to add a new custom [Vault secret plugin](https://www.vaultproject.io/docs/plugin) to retrieve the dataset credentials from where they are stored.
+
+
+## Configure Vault Kubernetes auth method in each cluster
+
```

Comment:
need to describe the prerequisite for k8s auth, such as Vault having connectivity to the remote cluster.

Reply:
OK, will add it.